user-docker/run_competitor_container.bash

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+CONTAINER="competitor-container"
```

Comment:
thanks! totally right, I did some last minute renaming... fixed in https://github.com/osrf/ariac-docker/pull/1/commits/b91d0dba49210be7a417fdf2bb28415a513d420f